### PR TITLE
Node and edge description tooltips

### PR DESCRIPTION
**Pull request by anthrocypher**
Adds tooltip showing edge and node type descriptions to weigh configuration sliders in SourceCred UI. Addresses issue #807. (The diff will make more sense once the revert in #1079 is approved and merged.)

Test plan:

Start SourceCred, open the weight config, and mouse over a node type name. Confirm that you see a tooltip describing that node type. Do the same for an edge type and verify that there's a tooltip.

### CODE REVIEW

**Comment by decentralion** [comment]
The pull request is currently in a broken state (tests fail, and the total diff just removes the description field). I think this is because its base branch is master rather than https://github.com/sourcecred/sourcecred/pull/1080. Since #1080 is approved, you should be able to merge #1080, and then rebase this branch on top of master.

**Comment by anthrocypher** [comment]
This branch is now up to date with master. Ready for review. cc: @decentralion

**Comment by anthrocypher** [comment]
Good lookin' out @decentralion. Test plan above has been updated, I've added unit tests for those three parameters, and there's an entry for this PR in CHANGELOG.md

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
 - Add the `export-graph` command (#1110)
 - Enable loading private repositories (#1085)
 - Enable setting type weights to 0 in the UI (#1005)
 - Add support for 🚀 and 👀 reaction types (#1068)
 - Create one page per project, rather than having a selector (#988)
-<!-- Please add new entries to the _top_ of this section. -->
+  <!-- Please add new entries to the _top_ of this section. -->
 
 ## [0.2.0]
+
```
Comment (decentralion) [inline CHANGELOG.md]: There shouldn't be extra newlines in the older sections of this document

Comment (BrianLitwin) [reply CHANGELOG.md]: Prettier makes these spacing changes to our Changelog file automatically when I save. Including the space beside the `<!--`. Took me a sec to figure out.

Comment (decentralion) [reply CHANGELOG.md]: Ah, I see. I have prettier configured to only run on js files locally. If prettier does this, then it's ok, I try never to argue with automatic code formatters.

**Pull request merged**